src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState player_states = 2;
+    required int32 num_development_cards_left = 3;
```

Comment:
thoughts on `remaining_developement_cards`? (I'm not huge on prefixing things with `num` because it's implied by the typing.)

Reply:
thats fine